resources/models/resource.py

```diff
@@ -86,6 +99,7 @@ class Resource(ModifiableModel, AutoIdentifiedModel):
     max_period = models.DurationField(verbose_name=_('Maximum reservation time'), null=True, blank=True)
 
     slug = AutoSlugField(populate_from=get_translated_name, unique=True)
+    equipments = models.ManyToManyField(Equipment, verbose_name=_('Equipments'), through='ResourceEquipment')
```

Comment:
English nitpick: `equipment` is both singular and plural, so even if this is a m2m field, its name should be `equipment`. :D